[Infrastructure] Build fails when using `make`

It seems that the table-gen files might be missing.

```
In file included from /home/dinistro/mlir-dev/thesis/lib/Conversion/StreamToHandshake/StreamToHandshak
e.cpp:17:
/home/dinistro/mlir-dev/thesis/include/circt-stream/Dialect/Stream/StreamDialect.h:14:10: fatal error:
 
      'circt-stream/Dialect/Stream/StreamOpsDialect.h.inc' file not found
#include "circt-stream/Dialect/Stream/StreamOpsDialect.h.inc"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

--- a/lib/Conversion/StreamToHandshake/CMakeLists.txt
+++ b/lib/Conversion/StreamToHandshake/CMakeLists.txt
@@ -13,5 +13,6 @@ add_mlir_library(CIRCTStreamStreamToHandshake
   MLIRFuncDialect
 
   CIRCTHandshake
+  CIRCTStreamStream
   )
 

